Matchfile

```diff
@@ -0,0 +1,13 @@
+git_url("https://github.com/jhass/insporation-certificates")
+
+storage_mode("git")
+
+type("development") # The default type, can be: appstore, adhoc, enterprise or development
+
+app_identifier(["eu.jhass.insporation"])
+username("thorstenclaus@web.de") # Your Apple Developer Portal username
```

Comment:
Should we really set this here? Given we don't really have a shared account, maybe better to let every user set it?

Reply:
Nope - Should use the shared developer account; During my local tests - this failed.